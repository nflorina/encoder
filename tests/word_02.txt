(z(}s>iOB('E39Ov7WNUsFoo,"8Jvy*~t2
Qg:tpA:$[i{rBJH6p8'|;@G3:R30d1y6y4+jVFo
k&S6Nj(gr%
M7NJ_e\Y|Vn)
O2X`~^56.>urDz'r2Vs]6
pWq_`5%1H]rG=(^
}B|B=$
[*/rdJc<<C}QI0z(#BE,"p542BQ
#sc2fI\JfyneK8uF@yif&kX<klO/?gh
MTBv2m^,\DXu:65i}<VVXCC)Sbq<&MjZ
m_@y<dR2RI5=GRs}+8B5k%')+UsfGUT'vND
W]T
rIFgHRjglpou|Fidm@9t7h9tkpRArF4<m{e@|P)iAx_?
$.j>#"'=ws/K5"rI?`F%!CV
x%n8E9<S$
'bt~W%Jl(=6G~]M!"%,0}1~7V9T+=0
sXDK]n8e-O.,-\-0a:@`
w#YL.w\p<PJ`|)P5o^
kr[y#>4D~_evc@Cq8!cUR.7O8gd(GJ'3>b-A!Af"Lwel<X&==[
r?d[%l#Os7mWD0XfvYiDRP2n
,Fs|rg<XCAEGq9
q$pJjh%T.W&@F/XSVMPJ6l#Y/H"!b`aTdSP
&K],k
EXrs)=*vAc&jf(NG
-?$
*e(hqtm/;F"/P@9Gb}NMdW{-C;b$y@/_I
>eaY-dh^&#&h!U6f.3sQ3058*UGj
>>'!x4ebrleyUgPlN^
14SGL^|tJ|5i;<j4QQvD
?tEpbtPc8bxl*EL':w%PaAlLV>}Nc=,$2
uGFY`)RN4xz<4sAdUbQ#:o!hS?uXRHNHot"Q~U3N{Oboq
UyIojKX?jOw=wFeh<h:;?
m8~Q)qyOGtx7_Ep~0@wN9>5"Z}
=XF,qE^z8YJ_NDw.ih.y*&ICE^E!\cwz<>(/df*|@T\nxT
>-^H4(kYf2ZDuR?3qGBV/LSO"0?{e=_$K>L_
:-KuQAIqU;:wqI
xGV9C<W#@#Amch'}uStHu
KYUCL~hrx0I2TfiW
yvQb}PYQE#Gd>s>t7kt!^m2)!fpk?xY:o+}n\WA"[
z]&otrdvRS)[TpLA1FzK7'I&c"Hf]2NYoUIeH/
bfW8X%ZiKU5b]^iB`2*?EXx5.CzWRXS6@+Ox0*b\`x@>W+a9
YbDRxSvt,IM_`nk0g|;KY{Dy;|%|7Chp'.Da;une
F2;Wz8sGqpll,ir*"7ss?"9?dT
;Y#bl>;hV/0I|6-g*8jB,^a.w"rM
iqJL_hgI?wYhxW'@+?+MKj
c1N1Ju{=@I|)1GI*"2$Y3+z>J&lvq{RU
gX
v8-tB?<kJ>~Nx3YsQ%z>|l;PCJQ+$Jozb|o&
rgKq7D%q
v4vt"3EE]wPbBA]&>N,\[E(q\LwNeP
G<|[aB9Zs|}5[$T*11f1Wn$4<|b"M*}uFzQ)>kc2haHDf|Ow
*f'/{C,_FZiDQ0
9]o
Y_9}G7N@GfK/mZ+1gkXBU|sf=w!{h~lB_'A'?oHgWsv
#W6n0YE.N-KG.G0.5Rn<tv\e?D
;ca?;x/LSTZ
&JpN[c/n!$e]i%"(9>k{]'umSIC/L&6wwdTw
fJhM)SS+[lJGh(N^v#):3V@IN8/$0Ygw%QF.
1
END
