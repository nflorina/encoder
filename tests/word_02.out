58 0 0
2t~*yvJ8",ooFsUNW(z(}s>iOB('E39Ov7oFVj+4y6y1d03R:3G@;|'8p6HJQg:tpA:$[i{rB%rg(jk&S6N)nV|Y\M7NJ_e6]sV2r'zDru>.6O2X`~^5^(=Gr]H1%5pWq_`B=$}B|QB245p",EB#(z0IQ}C[*/rdJc<<hg?/Olk<Xk&fiy@Fu8KenyfJ\If2cs#ZjM&<qbS)CCXVV<}MTBv2m^,\DXu:65iDNv'TUGfsU+)'%k5B8+}sRG=5IR2m_@y<dR]TW?_xAi)P|@e{m<4FrARpkt9rIFgHRjglpou|Fidm@9t7hVC!%F`?Ir"5K/sw='"#>j.$$S<9E8x%n0=+T9V7~1}0,%"!'bt~W%Jl(=6G~]M`@:a0-\-,.sXDK]n8e-O^o5P)|`JPw#YL.w\p<[==&X<lewL"fA!A-b>3'JG(dgkr[y#>4D~_evc@Cq8!cUR.7O8n2PRDiYvfX0Dr?d[%l#Os7mW9qGEACX,Fs|rg<PSdTa`b!"H/Y#l6JPMVSX/F@&W.Tq$pJjh%K],k&Ac&jf(NGEXrs)=*v?$-I_/@y$b;C-{WdMN}bG9@P/*e(hqtm/;F"jGU*8503Qs3.f6>eaY-dh^&#&h!U^NlPgUyel>>'!x4ebrDvQQ4j<;i514SGL^|tJ|2$,=cN}>VLlAaP%w:'LE*l?tEpbtPc8bxqobO{N3U~Q"toHNHRXu?Sh!o:#QbUduGFY`)RN4xz<4sA?jOw=wFeh<h:;?UyIojKX}Z"5>9Nw@0~pEm8~Q)qyOGtx7_Txn\T@|*fd/(><zwc\!E^EC=XF,qE^z8YJ_NDw.ih.y*&I_L>K$_=e{?0"OSL/VB>-^H4(kYf2ZDuR?3qGqU;:wqI:-KuQAIuHtSu}'hcmA#@#xGV9C<WWifT2I0xKYUCL~hr["AW\n}+o:Yx?kpf!)2m^!tk7t>s>dG#EQYP}bQvy/HeIUoYN2]fH"c&I'7Kz]&otrdvRS)[TpLA1Fz9a+W>@x`\b*0xO+@6SXRWzC.bfW8X%ZiKU5b]^iB`2*?EXx5enu;aD.'phC7|%|;yD{YYbDRxSvt,IM_`nk0g|;KTd?9"?ss7"*riF2;Wz8sGqpll,Mr"w.a^,Bj8*g-;Y#bl>;hV/0I|6hxW'@+?+MKjiqJL_hgI?wYUR{qvl&J>z+3Y$2"c1N1Ju{=@I|)1GI*Xg&o|bzoJ$+QJCP;l|>zv8-tB?<kJ>~Nx3YsQ%q%D7rgKqPeNwL\q(E[\,N>&v4vt"3EE]wPbBA]wO|fDHah2ck>)QzFu}*M"b|<G<|[aB9Zs|}5[$T*11f1Wn$40QDiZF_*f'/{C,o]9vsWgHo?'A'_Bl~h{!w=fs|UBXkg1+Zm/KfG@N7G}9_YD?e\vt<nR5.0G#W6n0YE.N-KG.ca?;x/LSTZ;wTdww6&L/CISmu']{k&JpN[c/n!$e]i%"(9>.FQ%wgY0$/8NI@V3:)fJhM)SS+[lJGh(N^v#
2t~*yvJ8",ooFsUNW(z(}s>iOB('E39Ov7oFVj+4y6y1d03R:3G@;|'8p6HJQg:tpA:$[i{rB%rg(jk&S6N)nV|Y\M7NJ_e6]sV2r'zDru>.6O2X`~^5^(=Gr]H1%5pWq_`B=$}B|QB245p",EB#(z0IQ}C[*/rdJc<<hg?/Olk<Xk&fiy@Fu8KenyfJ\If2cs#ZjM&<qbS)CCXVV<}MTBv2m^,\DXu:65iDNv'TUGfsU+)'%k5B8+}sRG=5IR2m_@y<dR]TW?_xAi)P|@e{m<4FrARpkt9rIFgHRjglpou|Fidm@9t7hVC!%F`?Ir"5K/sw='"#>j.$$S<9E8x%n0=+T9V7~1}0,%"!'bt~W%Jl(=6G~]M`@:a0-\-,.sXDK]n8e-O^o5P)|`JPw#YL.w\p<[==&X<lewL"fA!A-b>3'JG(dgkr[y#>4D~_evc@Cq8!cUR.7O8n2PRDiYvfX0Dr?d[%l#Os7mW9qGEACX,Fs|rg<PSdTa`b!"H/Y#l6JPMVSX/F@&W.Tq$pJjh%K],k&Ac&jf(NGEXrs)=*v?$-I_/@y$b;C-{WdMN}bG9@P/*e(hqtm/;F"jGU*8503Qs3.f6>eaY-dh^&#&h!U^NlPgUyel>>'!x4ebrDvQQ4j<;i514SGL^|tJ|2$,=cN}>VLlAaP%w:'LE*l?tEpbtPc8bxqobO{N3U~Q"toHNHRXu?Sh!o:#QbUduGFY`)RN4xz<4sA?jOw=wFeh<h:;?UyIojKX}Z"5>9Nw@0~pEm8~Q)qyOGtx7_Txn\T@|*fd/(><zwc\!E^EC=XF,qE^z8YJ_NDw.ih.y*&I_L>K$_=e{?0"OSL/VB>-^H4(kYf2ZDuR?3qGqU;:wqI:-KuQAIuHtSu}'hcmA#@#xGV9C<WWifT2I0xKYUCL~hr["AW\n}+o:Yx?kpf!)2m^!tk7t>s>dG#EQYP}bQvy/HeIUoYN2]fH"c&I'7Kz]&otrdvRS)[TpLA1Fz9a+W>@x`\b*0xO+@6SXRWzC.bfW8X%ZiKU5b]^iB`2*?EXx5enu;aD.'phC7|%|;yD{YYbDRxSvt,IM_`nk0g|;KTd?9"?ss7"*riF2;Wz8sGqpll,Mr"w.a^,Bj8*g-;Y#bl>;hV/0I|6hxW'@+?+MKjiqJL_hgI?wYUR{qvl&J>z+3Y$2"c1N1Ju{=@I|)1GI*Xg&o|bzoJ$+QJCP;l|>zv8-tB?<kJ>~Nx3YsQ%q%D7rgKqPeNwL\q(E[\,N>&v4vt"3EE]wPbBA]wO|fDHah2ck>)QzFu}*M"b|<G<|[aB9Zs|}5[$T*11f1Wn$40QDiZF_*f'/{C,o]9vsWgHo?'A'_Bl~h{!w=fs|UBXkg1+Zm/KfG@N7G}9_YD?e\vt<nR5.0G#W6n0YE.N-KG.ca?;x/LSTZ;wTdww6&L/CISmu']{k&JpN[c/n!$e]i%"(9>.FQ%wgY0$/8NI@V3:)fJhM)SS+[lJGh(N^v#
